x/wasm/simulation/proposals.go

```diff
@@ -146,6 +149,8 @@ func SimulateStoreCodeProposal(wasmKeeper WasmKeeper) simtypes.ContentSimulatorF
 }
 
 // Simulate instantiate contract proposal
+//
+//nolint:staticcheck
```

Comment:
This and others: no need to deactivate the linter. 
This function should return `sdk.Msg` instead. This is how [bank](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/x/bank/simulation/proposals.go) looks like for example

Reply:
Looks like we need to support old and new proposal types for some time. There will be a new PR to add new proposal types to  sim